README.md

```diff
@@ -21,7 +21,8 @@ We run it on Raspbery Pis (ARMv8) as a part of [Umbrel OS](https://github.com/ge
 
 If you're looking to run Umbrel on:
 
-- A Raspberry Pi 3 or 4 (recommended) - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os)
+- A Raspberry Pi 4 (recommended) - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os)
```

Comment:
I think you should also add something like this, some people might have an older 1GB version.

```suggestion
- A Raspberry Pi 4 with at least 2GB of RAM (recommended) - [Download Umbrel OS](https://github.com/getumbrel/umbrel-os)
```

Reply:
Cool thanks for that

Reply:
I think maybe also drop `(recommended)` since we are no longer recommending one of two options.

Reply:
Thats actually a good point, I've amended that

Reply:
We show "recommended" in comparison to "anything else":

![image](https://user-images.githubusercontent.com/10330103/94897992-63326600-04ae-11eb-9eb7-afbdcba224f4.png)

Reply:
Ahh, yes, missed that.